src/main/java/ch/njol/skript/tests/runner/SecParse.java

```diff
@@ -0,0 +1,83 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.tests.runner;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.Section;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.log.LogEntry;
+import ch.njol.skript.log.RetainingLogHandler;
+import ch.njol.skript.log.SkriptLogger;
+import ch.njol.util.Kleenean;
+import com.google.common.collect.Iterables;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.List;
+
+
+public class SecParse extends Section {
+
+	static {
+		Skript.registerSection(SecParse.class, "parse");
+	}
+
+	@Nullable
+	public static String[] lastLogs;
```

Comment:
You probably want to use lists, prevent modifying from the outside and ensure some kind of thread safety.

Something like this:
```java
private static final List<String> LAST_LOGS = Collections.synchronizedList(new ArrayList<>());

// modify
synchronized (LAST_LOGS) {
	LAST_LOGS.clear();
	LAST_LOGS.addAll(logs);
}

// getter
@Unmodifiable
public static List<String> lastLogs() {
	return Collections.unmodifiableList(LAST_LOGS);
}
```

Reply:
I'm not sure if we have to worry about thread safety here since this is the testing suite :)